notebooks/nessie-iceberg-flink-demo-nba.ipynb

```diff
@@ -0,0 +1,719 @@
+    "To get started, we will first have to do a few setup steps that give us everything we need\n",
+    "to get started with Nessie. In case you're interested in the detailed setup steps for Flink, you can check out the [docs](https://projectnessie.org/tools/flink/)\n",
+    "\n",
+    "The Binder server has downloaded flink and some data for us as well as started a Nessie server in the background. All we have to do is start Flink\n",
+    "\n",
+    "The below cell starts a local Flink session with parameters needed to configure Nessie. Each config option is followed by a comment explaining its purpose.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "from pyflink.datastream import StreamExecutionEnvironment\n",
+    "from pyflink.table import StreamTableEnvironment\n",
+    "from pyflink.table.expressions import lit\n",
+    "from pynessie import init\n",
+    "\n",
+    "# where we will store our data\n",
+    "warehouse = os.path.join(os.getcwd(), \"flink-warehouse\")\n",
+    "# this was downloaded when Binder started, its available on maven central\n",
+    "iceberg_flink_runtime_jar = os.path.join(os.getcwd(), \"../iceberg-flink-runtime-0.12.0.jar\")\n",
+    "\n",
+    "env = StreamExecutionEnvironment.get_execution_environment()\n",
+    "env.add_jars(\"file://{}\".format(iceberg_flink_runtime_jar))\n",
+    "table_env = StreamTableEnvironment.create(env)\n",
+    "\n",
+    "nessie_client = init()\n",
+    "\n",
+    "def create_ref_catalog(ref):\n",
+    "    \"\"\"\n",
+    "    Create a flink catalog that is tied to a specific ref.\n",
+    "\n",
+    "    In order to create the catalog we have to first create the branch\n",
+    "    \"\"\"\n",
+    "    hash_ = nessie_client.get_reference(nessie_client.get_default_branch()).hash_\n",
+    "    try:\n",
+    "        nessie_client.create_branch(ref, hash_)\n",
+    "    except:\n",
+    "        pass # already created\n",
+    "    # The important args below are:\n",
+    "    # type: tell Flink to use Iceberg as the catalog\n",
+    "    # catalog-impl: which Iceberg catalog to use, in this case we want Nessie\n",
+    "    # uri: the location of the nessie server.\n",
+    "    # ref: the Nessie ref/branch we want to use (defaults to main)\n",
+    "    # warehouse: the location this catalog should store its data\n",
+    "    table_env.execute_sql(\n",
+    "            f\"\"\"CREATE CATALOG {ref}_catalog WITH (\n",
+    "            'type'='iceberg',\n",
+    "            'catalog-impl'='org.apache.iceberg.nessie.NessieCatalog',\n",
+    "            'uri'='http://localhost:19120/api/v1',\n",
+    "            'ref'='{ref}',\n",
+    "            'warehouse' = '{warehouse}')\"\"\"\n",
+    "        )\n",
+    "create_ref_catalog(nessie_client.get_default_branch())\n",
+    "print(\"Flink running\")"
```

Comment:
Wonder whether it's possible to highlight this `Flink running` message. Maybe with one or two newlines before and after?

Current output is 
```
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/srv/conda/envs/flink-demo/lib/python3.7/site-packages/pyflink/lib/slf4j-log4j12-1.7.25.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/srv/conda/envs/flink-demo/lib/python3.7/site-packages/pyflink/lib/log4j-slf4j-impl-2.12.1.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.impl.Log4jLoggerFactory]
log4j:WARN No appenders could be found for logger (org.apache.flink.configuration.GlobalConfiguration).
log4j:WARN Please initialize the log4j system properly.
log4j:WARN See http://logging.apache.org/log4j/1.2/faq.html#noconfig for more info.
```
(the above w/ a red background) followed by
```
Flink running
```

Reply:
added some new lines. Its annoyingly difficult to color the font or change its size. Same w/ getting rid of the java logs

Reply:
to be more specific, its possible to change the output but it is verbose and detracts from the demo